Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -2128,14 +2150,50 @@ def get_asn_takedown_reputation(self, asn: str, explain: int = 0, limit: int = N
             # Log or wrap the unexpected response type for debugging or display
             return {"error": response if isinstance(response, str) else "Unexpected response type"}
 
-    def get_ipv4_reputation(self, ipv4: str, explain: int = 0, limit: int = None) -> dict[str, Any]:
+    def get_ipv4_reputation(self, ipv4: str, explain: int = 1, limit: int = None) -> dict[str, Any]:
         """
         Retrieve reputation information for an IPv4 address.
+
+        Args:
+            ipv4: The IPv4 address to check.
+            explain: Whether to include explanation details (1 for yes, 0 for no).
+            limit: Maximum number of history entries to return.
+
+        Returns:
+            Dictionary containing reputation information.
         """
         url_suffix = f"{IPV4_REPUTATION}/{ipv4}"
-        query_params = assign_params(limit=limit, explain=explain)
+        query_params = {}
+
```

Comment:
The function has too many return statements, can you restructure the code to have a single return. Also put some more logs in the function.